vtex/admin-ui
# Fix Sidebar second level height overflow

#### What is the purpose of this pull request?

As the title says.

It also adjust the zIndices applied on the sidebar

#### What problem is this solving?

Sidebar second level wasn't with scroll when there are several items on it. This PR fixes it.

#### How should this be manually tested?


Go to the seventh section on the sidebar and check that the scroll appears when there are many items.

#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
